model/span.go

```diff
@@ -50,6 +57,14 @@ var toSpanKind = map[string]trace.SpanKind{
 	"internal": trace.SpanKindInternal,
 }
 
+var toSamplerType = map[string]SamplerType{
+	"unknown":       SamplerTypeUnrecognized,
+	"probabilistic": SamplerTypeProbabilistic,
+	"lowerbound":    SamplerTypeLowerBound,
+	"ratelimiting":  SamplerTypeRateLimiting,
+	"const":         SamplerTypeConst,
+}
```

Comment:
you may also want to define `func (t SamplerType) String() string` function, since it's needed in cmd/collector/app/metrics.go